target/linux/mpc85xx/config-5.10

```diff
@@ -245,15 +245,17 @@ CONFIG_SPI_MEM=y
 CONFIG_SRCU=y
 # CONFIG_STRIP_ASM_SYMS is not set
 # CONFIG_STX_GP3 is not set
-CONFIG_SWCONFIG=y
```

Comment:
It seems that Ocedo Panda also has a built-in switch. Shouldn't swconfig remain enabled for the P1020 subtarget?

Reply:
Thank You. I forgot commit 'target/linux/mpc85xx/p1020/config-default' file.